contracts/JBController.sol

```diff
@@ -435,7 +458,9 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     JBFundingCycle memory _fundingCycle = fundingCycleStore.currentOf(_projectId);
 
     // The current funding cycle must not be paused.
-    require(_fundingCycle.changeTokenAllowed(), '0x05: NOT_ALLOWED');
+    if (!(_fundingCycle.changeTokenAllowed())) {
```

Comment:
I see this globally over this PR. Please always expand the negation. We should never had !(some condition(s))